docs/initial-set-up/users-contacts.md

```diff
@@ -1,5 +1,9 @@
 # How are Drupal or Joomla Users Linked to CiviCRM Contact Records
 
+Every CiviCRM instance sits within a "User Framework" &mdash; a web application that handles user authentication (logging in) and gives users various levels of access to CiviCRM features (permissions). Four different Content Management Systems (CMSs) can act as the User Framework for CiviCRM: Backdrop, Drupal, Joomla and WordPress.
```

Comment:
```suggestion
Every CiviCRM instance sits within a "User Framework" - a web application that handles user authentication (logging in) and gives users various levels of access to CiviCRM features (permissions).
```
Remove the HTML encoded dash (our markdown parser doesn't like these very much!) and remove the CMS discussion - it doesn't belong on this page and I'd like to avoid duplicating - we have a whole page covering CMSs and their relation to CiviCRM.